raw_data/preprocessed/africa/liberia_english.txt

```diff
@@ -0,0 +1,376 @@
+
+Designation of Analytical Laboratories, Analysts and Reference Analysts
+(96) The Agency may by published notice designate: [a] laboratories as it deems necessary and fit, to be analytical laboratories and reference laboratories for the purposes of this Law and shall specify; and [b] the notice shall specify who shall serve as analysts or reference analysts for the purposes of this Law.
+
+Certificate of Analysis
+(97) [1] A laboratory designated as an analytical or reference laboratory under this Law shall issue a certificate of analysis on a substance submitted to it pursuant to this Law; [2] The certificate of analysis shall state the methods of analysis followed and be signed by the analyst or reference analyst; [3] A certificate issued pursuant to subsection [1] shall be sufficient evidence of the facts stated in the certification commits an offence and is liable on conviction to a fine not exceeding 5,000 US DOLLARS or imprisonment for a period not exceeding one year or to both.
```

Comment:
Please replace each semi-colon preceding a number in square brackets with a period.